generic/tests/cfg/ethtool.cfg

```diff
@@ -1,15 +1,20 @@
 - ethtool:
     virt_test_type = qemu libvirt
     only Linux
-    only e1000, virtio_net
+    only e1000, e1000e, virtio_net
     no RHEL.3
     type = ethtool
     filesize = 512
     nic_mode = tap
     e1000:
```

Comment:
This could be `e1000, e1000e:` since they are totally same.

Reply:
@vivianQizhu Updated. Please help review again. Thanks.